cypress/integration/pages/index.js

```diff
@@ -0,0 +1,120 @@
+import config from '../../support/config/services';
+import envConfig from '../../support/config/envs';
+import appConfig from '../../../src/app/lib/config/services';
+import describeForEuOnly from '../../support/describeForEuOnly';
+
+const alwaysTests = ({ service, pageType }) => {
```

Comment:
I'm not sure about the naming of this. Does it run common tests? Maybe something like `runCommonTests` would give a better hint to readers?